komorebi/src/ring.rs

```diff
@@ -76,4 +76,30 @@ macro_rules! impl_ring_elements {
             }
         }
     };
+    ($name:ty, $element:ident, $el_name:literal) => {
```

Comment:
Can you add a comment here for future me to understand how/why the literal is used here, I figured it out reading this PR but I know I'm gonna forget in a few months